README.md

```diff
@@ -74,6 +75,14 @@ Each package is named with the `@deriv/` prefix, however for the scripts above,
 You can find the names of packages by first navigating to the `packages` folder. Each subfolder is a package, and contains a `package.json` file. The value of the `name` key in `package.json` is the package name.
 
 ### Usage
+#### Configuring Hosts file
+
+In order to run our solution for the first time, you need to configure your `hosts` file:
+1. Open terminal.
+2. Open `hosts` file in your preferred text editor, f.e `sudo vim /etc/hosts`.
+3. Change `127.0.0.1    localhost` entry to `127.0.0.1     localhost.binary.sx`.
```

Comment:
We can append the entry instead of changing current one. Right? So better to edit this.